harness/determined/cli/config_policies.py

```diff
@@ -0,0 +1,158 @@
+from determined.cli import render
+from determined.common import api, util
+from determined.common.api import bindings
+
+
+def describe_config_policies(args: argparse.Namespace) -> None:
+    sess = cli.setup_session(args)
+    workload_type = ""
+    if args.workload_type.upper() == "EXPERIMENT":
+        workload_type = "EXPERIMENT"
+    elif args.workload_type.upper() == "NTSC":
+        workload_type = "NTSC"
+    else:
+        raise cli.errors.CliError(
+            "Failed to list config policies: Invalid workload type provided."
+            + " Valid options: 'experiment' and 'ntsc'"
+        )
+
+    wksp = api.workspace_by_name(sess, args.workspace_name)
+    resp = bindings.get_GetWorkspaceConfigPolicies(
+        sess, workloadType=workload_type, workspaceId=wksp.id
+    )
+    if args.json:
+        render.print_json(resp.configPolicies)
+    else:
+        print(util.yaml_safe_dump(resp.configPolicies, default_flow_style=False))
+    return None
+
+
+def set_config_policies(args: argparse.Namespace) -> None:
+    sess = cli.setup_session(args)
+    try:
+        with open(args.path, "r") as f:
+            data = f.read()
+    except Exception as e:
+        raise cli.errors.CliError(f"Error opening file: {e}")
+    workload_type = ""
+    if args.workload_type.upper() == "EXPERIMENT":
+        workload_type = "EXPERIMENT"
+    elif args.workload_type.upper() == "NTSC":
+        workload_type = "NTSC"
+    else:
+        raise cli.errors.CliError(
+            "Failed to set config policies: Invalid workload type provided."
+            + " Valid options: 'experiment' and 'ntsc'"
+        )
+
+    wksp = api.workspace_by_name(sess, args.workspace_name)
+    body = bindings.v1PutWorkspaceConfigPoliciesRequest(
+        workloadType=workload_type, configPolicies=data, workspaceId=wksp.id
+    )
+    resp = bindings.put_PutWorkspaceConfigPolicies(
+        sess, workloadType=workload_type, workspaceId=wksp.id, body=body
+    )
+    print(util.yaml_safe_dump(resp.configPolicies, default_flow_style=False))
+    return None
+
+
+def delete_config_policies(args: argparse.Namespace) -> None:
+    sess = cli.setup_session(args)
+    workload_type = ""
+    if args.workload_type.upper() == "EXPERIMENT":
+        workload_type = "EXPERIMENT"
+    elif args.workload_type.upper() == "NTSC":
+        workload_type = "NTSC"
+    else:
+        raise cli.errors.CliError(
+            "Failed to delete config policies: Invalid workload type provided."
+            + " Valid options: 'experiment' and 'ntsc'"
+        )
+
+    wksp = api.workspace_by_name(sess, args.workspace_name)
+    bindings.delete_DeleteWorkspaceConfigPolicies(
+        sess, workloadType=workload_type, workspaceId=wksp.id
+    )
+    print(
+        f"Successfully deleted {workload_type} config policies for workspace {args.workspace_name}."
+    )
+    return None
+
+
+args_description: cli.ArgsDescription = [
+    cli.Cmd(
+        "config-policies",
```

Comment:
@kkunapuli @maxrussell what do you think about just policies? I remember deciding on config-policies to make it more discoverable and clear but I am open to either

Reply:
I think we should use "task config policies" in documentation as much as possible, but I'm fine with using just `policies` for the CLI command.

Reply:
I have two concerns with just "policy":
- I'm concerned that we'll have more than one kind of policy in our system and this will be convenient shorthand for now and then become confusing and ambiguous later.
- I'm also concerned that without a noun or verb to attach to "policy," users won't know what it's for and we'll have the discoverability problem Saloni mentioned. Log retention policy? Garbage collection policy? etc.

Reply:
that's fair. 

i was thinking about "policies" as a feature that would grow into something generic enough to encompass all sorts of system policies, or at least be the main "policy" feature that we support that it wouldn't be ambiguous. 

if all policies are attached to a config, in the future there might exist some parallel between existing CLI commands and their policies, like `det e set log-retention 1` and `det e policies set log-retention 1`. though the abstraction might not be as clean as i'm imagining.

i agree thought that right now just "policies" is kind of vague.

Reply:
That's a cool idea! Interesting to have it be kind of a subtree of the CLI that mirrors others.